File: python/ifTest.py

def check(a=''):
	if a == 'a':
		print('Apple')
	elif a == 'b':
		print('Banana')
	elif a == 'o':
		print('Orange')
	else:
		print('others')

def test():
	check('a')

def test2():
	check('b')

def test3():
	check('o')

def test4():
	check()

def test5():
	a=['Apple','Banana','Orange']
	if 'Apple' in a:
		print('Yes, I would like one')
	else:
		print("No, thanks")

def test6():
	if 'a' in "Banana":
		print("Yes")
	else:
		print("No")

test6()

File: kotlin/ifTest.kt

fun test(){
	val a=true
	if(a){
		println("apple")
	}else{
		println("banana")
	}
}

fun test2(){
    var a = 5
    var b = 3
    var c = if(a > b) "apple" else "banana"
    println(c)
}

fun test3(){
	val a=3
	if(a in 1..9){
		println("apple")
	}
}
